tests/schemas/invalid/test_invalid_relationship_no_list_specified.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
So this schema ensures that a list key is specified but we should ensure that if a list is specified, then that value of the list is actually a valid list?. Because if I pass a non-exisitent list to the `list` the check passes.